frontend/src/components/ReviewForm/ReviewForm.jsx

```diff
@@ -0,0 +1,55 @@
+import { useContext } from "react";
+import { newReviewContext } from "../../providers/NewReviewProvider";
+import LandlordSection from "./LandlordSection";
+import PropertySection from "./PropertySection";
+import RatingsReviewSection from "./RatingsReviewSection";
+import axios from 'axios';
+import { useNavigate } from "react-router-dom";
+
+const ReviewForm = () => {
+  const navigate = useNavigate();
+  const { landlord, property, reviewRating } = useContext(newReviewContext);
+
+  const handleSubmit = (e) => {
+    const newUser = {
+      title: reviewRating.title,
```

Comment:
I don't think I did my git revert properly :P I don't know why my form values are not the old ones we agreed on, but at least we're on the same page now!